harness/determined/tensorboard/metric_writers/pytorch.py

```diff
@@ -12,7 +12,7 @@
 from torch.utils.tensorboard import SummaryWriter  # isort:skip
```

Comment:
I guess here we are technically also violating the Google style guide rule but importing tensorboard instead would force you to give an alias to either the `from determined import tensorboard` or the `from torch.utils import tensorboard`, and that's probably not worth it.